lib/csvlint/cli.rb

```diff
@@ -130,14 +142,15 @@ def return_error(message)
       exit 1
     end
 
-    def validate_csv(source, schema, dump, json, werror, verbose)
+    def validate_csv(source, schema, dump, format, werror, verbose)
       @error_count = 0
-
-      validator = if (json === true) or (verbose === false)
+      @start = Time.now
```

Comment:
All these new variables inside `validate_csv` can be local variables instead of instance variables. Although this convention is not really followed well in this project! :)